.ansible-lint

```diff
@@ -2,6 +2,7 @@
 profile: production
 extra_vars:
   test_playbook: tests_default.yml
+  network_provider: nm
```

Comment:
This change needs a commit message that explains the reason for this and this should be a separate commit.

Reply:
I separated it into another commit, Thanks.